src/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { User } from '../types/User';
+
+type Props = {
+  userProps: User | null
+};
+
+export const UserInfo: React.FC<Props> = (props) => {
+  const { userProps } = props;
+
+  if (!userProps) {
+    return (
+      <li>
+        invalid data
+      </li>
+    );
+  }
+
+  const { name, email } = userProps;
+
+  return (
+    <>
+      <li className="list__user-name">
```

Comment:
```suggestion
      <p className="list__user-name">
```
It's not a totally semantically correct code. Use some other tags instead of pair of `li`s